resources/user-agents/browsers/safari/safari-generic.json

```diff
@@ -23,14 +23,14 @@
       },
       "children": [
         {
-          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Version/* Safari/*",
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit*(*khtml*like*gecko*)*Version/*Safari/*",
```

Comment:
For this one I didn't replace the space between the parenthesis and `applewebkit` with an asterisk because it caused these patterns to grab some others, like AdBeat bot.